src/_data/LiveData.js

```diff
@@ -58,6 +60,12 @@ export const changeLanguage = langCode => {
 };
 
 const initAuthorized = async (authData, store) => {
+    if (/japan/.test(authData.authorize.landing_company_name)) {
```

Comment:
We can just hide the trade panel for now.
Why not allow them to use all the rest?